### ISSUE TITLE
Code in a fenced block affects what is outside the block.

### ISSUE THREAD

**Issue by seperman** [jonschlinkert/sublime-markdown-extended#196]
For example the `"""` in a block of python code leaks out:

![Screenshot from 2019-12-23 16-18-21](https://user-images.githubusercontent.com/2314797/71386059-0eb59e00-25a0-11ea-8f66-b1746418f6d2.png)

**Comment by voltechs** [comment]
Can confirm this is still an issue. I guess in closer inspection, `Top 100 Sublime Text Markdown Syntax Highlighter` isn't really saying much given the "Sublime Text Markdown Syntax Highlighter" qualifier :P

<img width="1513" alt="Screen Shot 2020-12-22 at 12 55 53" src="https://user-images.githubusercontent.com/1700514/102927876-08792c80-4455-11eb-8fd7-a20fb48a9420.png">

**Comment by dencoiq** [comment]
Bump. This issue is still present and wasn't fixed 😢

**Comment by markhilkert** [comment]
Yeah I'm weirdly having the same issue. But for some reason it's only affecting me for python, not for Ruby or Bash:

<img width="821" alt="image" src="https://user-images.githubusercontent.com/46462767/160037562-ba925f0e-234a-4be0-8eba-0c10d0aad50d.png">

**Comment by nestedfunction** [comment]
Bump. Same issue.

**Comment by abhishekgupta-ontic** [comment]
https://github.com/jonschlinkert/sublime-markdown-extended/issues/140

**Comment by abhishekgupta-ontic** [comment]
For now I have shifted to 
https://sublimetext-markdown.github.io/MarkdownEditing/
